### ISSUE TITLE
Approximate crack time is misleading

### ISSUE THREAD

**Issue by DurandA** [mike-hearn/useapassphrase#10]
Just my two cents, but I think the approximate crack time is misleading.

The entropy is currently `7776^4=3656158440062976` while _zxcvbn_ uses the passphrase as a random string:
https://github.com/mike-hearn/useapassphrase/blob/0aa28d4f93c6e760abd4aad972c496bfbddc85c0/js/script.js#L64

_zxcvbn_ uses a list of common words and passwords to account for passwords that are not randomly generated. This effectively reduces the `zxcvbn()` return value if the words selected are part of the _zxcvbn_ wordlist. Hint: there are 6700 words from the EFF wordlist out of 7776 that are parts of _zxcvbn_ wordlist. Very long `timeToCrack` indicates that the generated passphrase contain an EFF word that is not part of the _zxcvbn_ wordlist.

An attacker may have clues that a password was generated using this common EFF wordlist. While it is still mostly unfeasible to brute-force, I still think that this is slightly misleading to indicate disproportionate crack times.

**Comment by DurandA** [comment]
Note that I used this Python script to compute how many words are common between _zxcvbn_ and the EFF wordlist (converted into JSON):

```python3
import os, json

passwords = set()
path = './data'
for filename in os.listdir(path):
    with open(os.path.join(path, filename), 'r') as f:
        for password in f:
            passwords.add(password.split()[0])
with open('wordlist.json', 'r') as f:
    wordlist = set(json.load(f))
print(len(wordlist.intersection(passwords)))
```

**Comment by BrendanMitchellSisu** [comment]
Agreed. The zxcvbn estimate isn't really a good one because we must assume the attacker knows the pattern being used. If, for example, the Use A Passphrase tool becomes very commonly used for generating passwords, then this might well be the case. Using the same guesses-per-second assumption that is used on the webpage, I'd estimate a crack time of 60 centuries for any 4-word passphrase generated here.
